CyberwatchApi.psm1

```diff
@@ -57,11 +75,30 @@ Class CbwApiClient {
     }
 
     [object] request([string]$http_method, [string]$request_URI) {
-        return SendApiRequest -api_url $this.api_url -api_key $this.api_key -secret_key $this.secret_key -http_method $http_method -request_URI $request_URI
+        return SendApiRequest -api_url $this.api_url -api_key $this.api_key -secret_key $this.secret_key -http_method $http_method -request_URI $request_URI | ConvertFrom-Json
     }
 
     [object] request([string]$http_method, [string]$request_URI, [Hashtable]$content) {
-        return SendApiRequest -api_url $this.api_url -api_key $this.api_key -secret_key $this.secret_key -http_method $http_method -request_URI $request_URI -content $content
+        return SendApiRequest -api_url $this.api_url -api_key $this.api_key -secret_key $this.secret_key -http_method $http_method -request_URI $request_URI -content $content | ConvertFrom-Json
+    }
+
+    [object] get_all_pages([string]$last_page_number,[string]$request_URI, [Hashtable]$filter) {
+        $allPages = New-Object System.Collections.ArrayList @()
+        for ($i=0; $i -lt $last_page_number; $i++) {
+            $filter["page"] += 1
```

Comment:
si tu as un itérateur c'est bien de l'utiliser, tu devrais faire `$filter["page"] = $i`